lib/testLauncher/SuitestLauncher.js

```diff
@@ -27,12 +27,16 @@ const {captureException} = require('../utils/sentry/Raven');
 const texts = require('../texts');
 const {registerProcess} = require('./processReaper');
 const {config} = require('../../config');
+const {version} = require('../../package.json');
+const {fetchLatestSuitestVersion, warnNewVersionAvailable} = require('../utils/packageMetadataHelper');
 
 class SuitestLauncher {
 	constructor(ownArgv = {}, restArgs = []) {
 		this.ownArgv = ownArgv;
 		this.restArgs = restArgs;
+		this.latestSuitestVersion = '';
 
+		fetchLatestSuitestVersion().then(version => this.latestSuitestVersion = version);
```

Comment:
is it possible that request resolves later than suitest launches child processes?